stable/unifi/templates/discovery-svc.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "unifi.fullname" . }}-discovery
+  labels:
+    app: {{ template "unifi.name" . }}
+    chart: {{ template "unifi.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.discoveryService.type }}
+  ports:
+    - port: {{ .Values.discoveryService.port }}
+      targetPort: discovery
+      protocol: UDP
+      name: discovery
```

Comment:
As with the controller - if a specific nodePort is defined in values.yml for, nothing would pick that up as there is no check for it here.

Reply:
fixed!